docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/visionos/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "VisionOS tracking"
+sidebar_position: 110
+---
+
+# VisionOS tracking
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+:::note
+Snowplow VisionOS tracking was added in version 6.0.0.
+:::
+
+The Snowplow iOS tracker supports tracking within VisionOS apps. All the usual events, including screen views, can be tracked as for any SwiftUI app. We've also provided additional events and entities to help you understand your VisionOS users.
+
+The immersive space context entity can be semi-automatically added to all events.
+
+## VisionOS events
+
+The events are: `OpenWindowEvent`, `DismissWindowEvent`, `OpenImmersiveSpaceEvent`, and `DismissImmersiveSpaceEvent`.
+
+### Window events
+
+Track the opening and dismissing of SwiftUI window groups using `OpenWindowEvent` and `DismissWindowEvent`. These events can be used in any SwiftUI app, not just VisionOS. The event data is sent as a window group context entity (see below) attached to these events; the events themselves have no properties.
+
+#### OpenWindowEvent
+
+```swift
+let tracker = Snowplow.createTracker(namespace: "appTracker", endpoint: "https://snowplow-collector-url.com")
+
+let event = OpenWindowEvent(
+    id: "group 1", // window group ID
+    uuid: UUID(), // optional UUID
+    titleKey: "window 1", // optional window title
+    windowStyle: WindowStyle.automatic // optional style of the window group
+)
+tracker.track(event)
+```
+*Schema:*
```

Comment:
Could we create a page under the Out-of-the-box data page (e.g., "visionOS and SwiftUI") where people could go to look at the schemas (similar to this for [app lifecycle](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/ootb-data/mobile-lifecycle-events/))? Then we could link to it from here.